definitions/npm/@testing-library/react-hooks_v3.x.x/flow_v0.104.x-/react-hooks_v3.x.x.js

```diff
@@ -0,0 +1,70 @@
+// @flow
+// Initially adapted from: https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/testing-library__react-hooks.
+
+declare module '@testing-library/react-hooks' {
+  declare export type RenderHookOptions<P> = {|
+    +initialProps?: P,
+    +wrapper?: React$ComponentType<{|
+      ...P,
+      +children: React.Node,
```

Comment:
I guess you'd have to import React here for this notation to work? `React$Node` otherwise.

Reply:
Fixed 🙇 ! I think that was the last of it.